apps/courses/src/routes/courses/_helpers.ts

```diff
@@ -343,3 +344,25 @@ export const useCoursePermissionGate = (user, lessons, page, params) => {
 
   return { page, user, lessons, params };
 };
+
+export const userIsMentor = async (user: firebase.User) => {
```

Comment:
`react-hooks/rules-of-hooks` will complain about using hooks (useFirestore/useState/useEffect) inside this function due to the function being named `userIsMentor`. 

Having it as `useUserIsMentor` is... strange, in my opinion. As an improvement, we could have a hook that fetches the fs user. I've created issue #147 for this.